feat: add "About the Boring Education" section with logo and project link

POBOROJO opened the pull request:
This pull request includes changes to the `src/pages/webinar/[webinarSlug]/index.tsx` file to enhance the content and functionality of the Webinar page. The most important changes include adding new imports for components and constants, and introducing a new section about "The Boring Education" with relevant content and a call-to-action button.

Enhancements to the Webinar page:

* `src/pages/webinar/[webinarSlug]/index.tsx`: Added imports for `imageMeta` from `@/constant/global` and `LinkButton` from `@/components/common/Buttons/LinkButton`. ([src/pages/webinar/[webinarSlug]/index.tsxR16-R17](diffhunk://#diff-7968a24aa67c0c9f80428ac23643a9cbcd61936558128d0294e2d6f660633255R16-R17))
* `src/pages/webinar/[webinarSlug]/index.tsx`: Introduced a new section with a heading "About the Boring Education", an image, a descriptive text, and a `LinkButton` that directs users to explore free projects. ([src/pages/webinar/[webinarSlug]/index.tsxR250-R287](diffhunk://#diff-7968a24aa67c0c9f80428ac23643a9cbcd61936558128d0294e2d6f660633255R250-R287))

POBOROJO committed:
feat: add 'About the Boring Education' section with logo and project exploration link

imsks commented on src/pages/webinar/[webinarSlug]/index.tsx:
```diff
@@ -13,6 +13,8 @@ import { useApi, useUser } from '@/hooks';
 import { routes } from '@/constant';
 import { WebinarHeroContainer } from '@/components';
 import { isProgramActive } from '@/utils';
+import { imageMeta } from '@/constant/global';
+import LinkButton from '@/components/common/Buttons/LinkButton';
```
always use Root address like @/constant or @/components not @/components/common/Buttons/LinkButton

POBOROJO committed:
Merge remote-tracking branch 'upstream/release/v-0.8.0' into pobo-feat-webinar

POBOROJO committed:
refactor: update import statements in webinar page for consistency

POBOROJO commented:
Done

imsks committed:
Update index.tsx

POBOROJO committed:
refactor: adjust layout and styling for 'About the Boring Education' section in webinar page

POBOROJO committed:
Merge branch 'pobo-feat-webinar' of https://github.com/POBOROJO/TBE-Web-poborojo into pobo-feat-webinar

POBOROJO committed:
refactor: enhance logging and error handling in certificate generation process

POBOROJO committed:
refactor: update webinar API to use slug instead of webinarId for improved routing and consistency
and made the Certifcate display for the enrolled users on clicking Generate Certificate

imsks commented on src/constant/routes.ts:
```diff
@@ -37,7 +37,7 @@ const routes = {
     base: '/api/v1',
     projects: '/projects',
-    webinar: '/webinar',
+    webinar: '', // adding /webinar here will result in /api/v1/webinar/webinar => 404(not found)
     project: (project: string) => `/projects/${project}`,
```
We need this route. Why did you delete this

imsks commented on src/pages/webinar/[webinarSlug]/index.tsx:
```diff
@@ -68,14 +67,29 @@ const WebinarPage = ({
-      if (!response?.status) {
-        throw new Error(`Error occured`);
+      // Log the full response for debugging
+      console.log('Certificate generation response:', response);
```
Remove logs after development

imsks commented on src/pages/webinar/[webinarSlug]/index.tsx:
```diff
@@ -68,14 +67,29 @@ const WebinarPage = ({
   const onGenerateCertificate = async () => {
     try {
+      // Add more detailed logging
+      console.log('Generating Certificate for:', user?.email);
```
Here also

imsks commented on src/pages/webinar/[webinarSlug]/index.tsx:
```diff
@@ -68,14 +67,29 @@ const WebinarPage = ({
+        throw new Error(response.message || 'Certificate generation failed');
       }
 
+      // if (!response?.status) {
```
Remove commented code

POBOROJO committed:
refactor: update webinar API route and improve error logging in certificate generation and removed console logs

POBOROJO committed:
refactor: clean up .gitignore by removing unnecessary entries
